src/types.d.ts

```diff
@@ -0,0 +1,56 @@
+
+declare module '@google-cloud/common' {
```

Comment:
Instead of this, could we push a d.ts to the @google-cloud/common github repo?  Or create a @types definition?

Reply:
I followed this approach to be consistent with the approach taken in the trace agent.  Hopefully this file will be temporary, and I think it would be better to convert `@google-cloud/common` to Typescript than to just add a `d.ts` file to its repo.